src/main/default/lwc/caseAssistFlow/caseAssistFlow.js

```diff
@@ -114,6 +114,11 @@ export default class CaseAssistFlow extends LightningElement {
 
   // When a picklist value is selected.
   handlePicklistChange(event) {
+    // Ignore the picklist change event when it was generated by changing the value programmatically.
+    if (this.skipPicklistOnce === event.target.fieldName) {
```

Comment:
I'm concerned by the fact that the analytics client can't handle sending two events. Maybe I'm misunderstanding but wouldn't it be better if it just queued them so you wouldn't have to worry about this in the first place?

Reply:
I believe it would yes :( however that problem doesn't reside within this cookbook.
And in the end, it was just wrong to be sending 2 "click" events for 1 user action, even if the events were to work correctly.

Reply:
eh bin. all good then